### ISSUE TITLE
NVDA 2020.1 exits and immediately restarts when Firefox opens all mew windows after link selection

### ISSUE THREAD

**Issue by Justineg123** [nvaccess/nvda#11235]
I am a visually impaired user and not a developer, so will do my best to answer the questions below. But at present this issue is making NVDA unworkable and therefore I cannot use my computer  which is my lifeline to the outside world!
I have attached crash.dwp as zip file
[nvda_crash.zip](https://github.com/nvaccess/nvda/files/4742812/nvda_crash.zip)

### Steps to reproduce:
Using desktop PC running Windows 10. Open Firefox browser. Then Google search any thing. From results pages click any link. When the page opens NVDA exits and immediately restarts. With any web page open click any link on the website and the same issue occurs. I have the xounc on for exit and start for NVDA

### Actual behavior:
Clicking on any links   stops and restarts NVDA running. Occaisonally when restarted  the screen rader won’t talk and has to be restarted again by myself
## ## Expected behavior:
NVDA to run without  stopping and starting. By itself

### System configuration
2020.1
#### NVDA installed/portable/running from source:
installed. Auto updates switched on
#### NVDA version:
2020.1
#### Windows version:
Windows 10 Home version 1909
#### Name and version of other software in use when reproducing the issue:
Firefox  Browser version 77.0.1 (64-bit)
#### Other information about your system:
64 bit operating system
Installed RAM 8GB
Intel Celeron
Wired keyboard, wireless mouth, Bluetooth speaker
Auto updates all on for NVDA, Firefox and Windows
#### Does the issue still occur after restarting your computer?
Yes, every time
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No. I am only a user and do not know how to install a previous version.
#### If addons are disabled, is your problem still occuring?
There are no add ons installed
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes. Several times as it  runs for about 1 second and then dialog box appears and says "complete" with OK button!
It is so quick I am not sure it is actually working. But it makes no difference to my issue!

**Comment by XLTechie** [comment]
Have you tried any other browser, such as Brave or Chrome,?

**Comment by DrSooom** [comment]
If possible, open "about:support" in Firefox and search for "Accessibility". The values here must be as follows: true, 0, true and NVDA

**Comment by Brian1Gaff** [comment]
A few thoughts here, you might first try to download the add on nvda windows 
10 essentials, or similar first.

It will be interesting to see if this works.

I think I would suggest that you try narrator in windows 10 as well, as this 
might aid you in seeing what exactly is going on. I'll not repeat what 
others have suggested, but if you have edge or Chrome on the machine, you 
might want to try those browsers and even Internet Explorer if its 
installed. What we need are some pointers here. As you are a beginner its 
going to be difficult to ask you to send a log file of the right type.
 What other things are on this computer. IE does it have a third party anti 
virus on it? If so which one? Has this ever worked since you started with 
nvda, ie, has it happened suddenly or is it from the word go, so to speak?

I'd be surprised if its the accessibility settings as normally when this is 
wrong, nvda simply goes quiet. To make it reboot is a little surprising.
 I also note your machine has quite a low powered processor on it, but 
normally this would only result in slow operation.

If you go into the menu system of nvda, you might want to try creating a 
portable version  in a folder on the main drive, and see if  it still has 
the issue from that, but ideally a new clean install is to be preferred 
unless you can see the old log after it restarts, and make sure its set to a 
high level of logging before you try to create the issue.


If I may ask if its doing all this restarting what did you fill this 
template out with? Another machine being handy might help us a bit.

Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users

**Comment by Adriani90** [comment]
@Justineg123 are you sure that NVDA restarts? Do you hear the exit and the start sounds?
Can you also reproduce this issue without bluetooth speakers?

I cannot reproduce this issue with NVDA 2020.1 and Firefox 77.

**Comment by Justineg123** [comment]
Thanks everyone for all the advice. I tried all the suggestions and the last one from Adriani90   is the only one to work. I removed my Bluetooth speaker (Echo Dot) and connected wired speakers and it has been several hours but, so far, no issues at all!  Thank you so much
